Breeder/BR_EnvelopeUtil.cpp

```diff
@@ -1908,10 +1910,10 @@ WDL_FastString BR_Envelope::GetProperties ()
 		properties.Append(m_properties.paramType.Get());
 		properties.Append("\n");
 		properties.AppendFormatted(256, "ACT %d %d\n", m_properties.active, m_properties.AIoptions);
-		properties.AppendFormatted(256, "VIS %d %d %d\n", m_properties.visible, m_properties.lane, m_properties.visUnknown);
+		properties.AppendFormatted(256, "VIS %d %d %d\n", m_properties.visible, m_properties.lane, 1.0);
```

Comment:
```suggestion
		properties.AppendFormatted(256, "VIS %d %d %d\n", m_properties.visible, m_properties.lane, 1);
```

`1.0` must be an integer to match the `%d` format. Otherwise garbage is inserted into `properties`. I'll force-push a commit to fix this and amend the 2.12.1 branch with it.

```
../Breeder/BR_EnvelopeUtil.cpp:1933:94: warning: format specifies type 'int' but the argument has type 'double' [-Wformat]
                properties.AppendFormatted(256, "VIS %d %d %d\n", m_properties.visible, m_properties.lane, 1.0);
                                                           ~~                                              ^~~
                                                           %f
```